include/bluetooth/bluetooth.h

```diff
@@ -71,6 +79,36 @@ int bt_enable(bt_ready_cb_t cb);
  */
 int bt_set_id_addr(const bt_addr_le_t *addr);
 
+/** @brief Get the currently configured identities.
+ *
+ *  Returns an array of the currently configured identity addresses.
+ *
+ *  @param addrs Array where to store the configured identities.
+ *  @param count Should be initialized to the array size. Once the function
```

Comment:
Use an array with CONFIG_BT_ID_MAX entries.

Reply:
I realize that it's less useful to want to get anything else than this amount, but at least the API would allow getting the default identity (using one element array), if the app doesn't care for the others.

Reply:
Are you planning to have API allowing get the most/least used identity or iterate over identities in some predefined order?

Usecases:
- Direct advertising to multiple masters (starting from the most frequently used identity). 
- Remove the least used identity in order to make room for new.

Reply:
> Are you planning to have API allowing get the most/least used identity or iterate over identities in some predefined order?

I would leave this up to the application. General principle being: keep the API as simple as possible and only add new features to it if they're not possible to do with the existing API plus some extra logic on the application side.